### PR TITLE
Flatten login screen

### PR DESCRIPTION
**Pull request by ccoy**
This PR continues #1908 by creating a flatter visual style on the login screen. 

- The Kolide logo is updated to the vertical style

- Gradient button has been changed to a solid color, drop shadow removed

- Background image has been removed, replaced with solid white

- Additional color tweaks to match the main app

Screenshot attached for reference.

screen shot 2018-08-28 at 12 15 39 pm

### CODE REVIEW

File: frontend/components/AuthenticationFormWrapper/AuthenticationFormWrapper.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ class AuthenticationFormWrapper extends Component {
 
     return (
       <div className={baseClass}>
-        <img alt="Kolide text logo" src="/assets/images/kolide-logo-text.svg" className={`${baseClass}__logo`} />
+        <img alt="Kolide vertical logo" src="/assets/images/kolide-logo-vertical.svg" className={`${baseClass}__logo`} />
```
Comment (kyleknighted) [inline frontend/components/AuthenticationFormWrapper/AuthenticationFormWrapper.jsx]: The alt should just be `Kolide` or maybe `Kolide Fleet`

File: frontend/components/buttons/Button/_styles.scss [hunk 2]
```diff
@@ -128,18 +127,17 @@ $base-class: 'button';
...
     font-weight: $light;
     letter-spacing: 4px;
     padding: $medium;
     width: 100%;
     height: auto;
-    box-shadow: inset 0 -3px 0 0 rgba(0, 0, 0, 0.25), 0 -2px 3px 0 rgba(0, 0, 0, 0.15);
 
     &:active {
-      box-shadow: 0 1px 0 #734893, 0 -2px 0 #d1d9e9;
+      box-shadow: none;
```
Comment (kyleknighted) [inline frontend/components/buttons/Button/_styles.scss]: If you're removing the box-shadow from `--gradient` than setting it to "none" is not needed. You can remove this whole :active section.

File: frontend/components/forms/LoginForm/_styles.scss [hunk 3]
```diff
@@ -16,8 +15,8 @@
     @include display(flex);
     align-items: center;
     background-color: $white;
-    border-top-left-radius: 4px;
-    border-top-right-radius: 4px;
+    border-top-left-radius: 0;
```
Comment (kyleknighted) [inline frontend/components/forms/LoginForm/_styles.scss:18]: If they are never set from the get-go, no reason to set them to 0. Just remove.

File: frontend/components/forms/LogoutForm/_styles.scss [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
 
   &__container {
     @include display(flex);
     @include align-items(center);
     @include flex-direction(column);
     background-color: $white;
-    border-top-left-radius: 4px;
-    border-top-right-radius: 4px;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
```
Comment (kyleknighted) [inline frontend/components/forms/LogoutForm/_styles.scss]: Remove if not resetting.

File: frontend/components/forms/fields/InputField/_styles.scss [hunk 5]
```diff
@@ -1,7 +1,7 @@
 .input-field {
   @include transition(border 150ms ease-in-out, box-shadow 150ms ease-in-out);
   line-height: 34px;
-  border-radius: 2px;
+  border-radius: 0;
```
Comment (kyleknighted) [inline frontend/components/forms/fields/InputField/_styles.scss]: Remove?

File: frontend/components/forms/fields/InputFieldWithIcon/_styles.scss [hunk 6]
```diff
@@ -20,8 +20,8 @@
   }
 
   &__input {
-    border: 0;
-    border-bottom: 2px solid $brand-ultralight;
+    border: 2px;
```
Comment (kyleknighted) [inline frontend/components/forms/fields/InputFieldWithIcon/_styles.scss:23]: border: 2px what? Missing all the important bits for a border. Color, shape, etc.

File: frontend/pages/LoginSuccessfulPage/_styles.scss [hunk 7]
```diff
@@ -1,7 +1,6 @@
 .login-success {
   background-color: $white;
-  box-shadow: 0 5px 30px 0 rgba($black, 0.3);
-  border-radius: 4px;
+  border-radius: 0;
```
Comment (kyleknighted) [inline frontend/pages/LoginSuccessfulPage/_styles.scss]: Remove?

File: frontend/components/AuthenticationFormWrapper/AuthenticationFormWrapper.jsx (hunk 1 above)
Comment (sabrinabuckets) [reply frontend/components/AuthenticationFormWrapper/AuthenticationFormWrapper.jsx]: Changed

File: frontend/components/forms/fields/InputFieldWithIcon/_styles.scss (hunk 6 above)
Comment (sabrinabuckets) [reply frontend/components/forms/fields/InputFieldWithIcon/_styles.scss:23]: This actually sets the border for the text entry fields. I'm not entirely sure how it works, but it do.

**Pull request merged**